Card/src/main/java/com/paypal/android/card/Card.kt

```diff
@@ -0,0 +1,3 @@
+package com.paypal.android.card
+
+data class Card(val number: String, val expiry: String)
```

Comment:
Do we need an additional value for security code?

Reply:
the `confirm_payment_source` query accepts `number` and `expiry` and thats it. Im not sure what the endpoint does though, is it just validating that you can use the card, or is it actually "paying" the order?

Reply:
The `confirm-payment-source` endpoint will attach a card (or other payment source) to an orderId. 

If you look at the ppaas documentation for this call, there are additional optional params that can be passed into the Card object (`security_code`, `card_type` etc.). I think we'll need to at least pass security code to the API for the first iteration of the SDK.

Reply:
Rebased onto main so the Card object should now be up to date.